Remotion/Web/UnitTests/Core/Context/HttpContextStorageProviderWithCallContextBasedFallbackTest.cs

```diff
@@ -24,18 +24,19 @@
 
 namespace Remotion.Web.UnitTests.Core.Context
 {
+#if NETFRAMEWORK
   [TestFixture]
-  public class HttpContextStorageProviderTest
+  public class HttpContextStorageProviderWithCallContextBasedFallbackTest
   {
-    private HttpContextStorageProvider _provider;
+    private HttpContextStorageProviderWithCallContextBasedFallback _provider;
     private HttpContext _testContext;
 
     [SetUp]
     public void SetUp ()
     {
       _testContext = HttpContextHelper.CreateHttpContext("x", "y", "z");
       HttpContext.Current = _testContext;
-      _provider = new HttpContextStorageProvider();
+      _provider = new HttpContextStorageProviderWithCallContextBasedFallback();
     }
 
```

Comment:
please add a tear down that sets HttpContext.Current=null. That's missing in the test and since we're touching the code and global state is such a nasty thing, we shouldn't let this go unanswered.